v1.0/inference.py

```diff
@@ -0,0 +1,34 @@
+from fastai2.vision.all import *
+
+# REPLACE THIS MODEL WITH MOST UP TO DATE VERSION
+MODEL = 'rn50-448px-v0.9-export.pkl'
+
+
+def get_lbl(fname):
+    """return the label of given image"""
+    lbl = fname.parent.name
+    if lbl == 'P':
+        return LBL_PATHO
+    elif lbl == 'C':
+        return LBL_COV
+    else:
+        raise ValueError(f'could not extract label from {fname}')
+        
+
+def in_valid(fname):
+    """return True if file in validationset, else False"""
+    return fname.parent.parent.name == 'valid'
+
+
+LEARN = load_learner(MODEL)
+
+CATEGORIES = LEARN.dls.vocab # classes in order of output neurons
+
+
+def predict(img):
```

Comment:
please add an example and provide an example image somewhere in the repo (where we know that there is no copyright problem).

The example could look like adding a main() function here such as:

```python
if __name__ == "__main__":
    img='<insert the actual path here where the sample image from our repo resides, relative path names are great'
    print(f'classifying image {img}: ')
    print(predict(img))

Reply:
ahm, I meant please do supply an image really in the repo so that the repo is self contained and runs out-of the box.